### PR TITLE
[fbcnms-alarms]: Test alarms links

### PR DESCRIPTION
**Pull request by HannaFar**
Add a unit test for troubleshooting links

### CODE REVIEW

File: fbcnms-packages/fbcnms-alarms/components/rules/__tests__/LabelsEditor-test.js [hunk 1]
```diff
@@ -115,40 +115,6 @@ test('labels without a key are filtered out', () => {
     testKey1: 'testVal1',
   });
 });
-test('clicking the reset button resets the label form to its initial state', () => {
```
Comment (andreilee) [inline fbcnms-packages/fbcnms-alarms/components/rules/__tests__/LabelsEditor-test.js:118]: Why was this removed?

Comment (HannaFar) [reply fbcnms-packages/fbcnms-alarms/components/rules/__tests__/LabelsEditor-test.js]: there is no reset button

**Review by andreilee — APPROVED** [review]
